SU2_PY/SU2_Nastran/pysu2_nastran.py

```diff
@@ -190,7 +251,7 @@ def __init__(self, config_fileName, ImposedMotion):
     self.Config_file = config_fileName
     self.Config = {}
 
-    print("\n------------------------------ Configuring the structural tester solver for FSI simulation ------------------------------")
+    print("\n---------- Configuring the structural tester solver for FSI simulation ----------")
```

Comment:
It would be better to use rfill / lfill with the total width defined somewhere central.

Reply:
Great! Thank you for the suggestion